galaxyproject/galaxy
# Implement datatype upload warnings

To push people away from potentially problematic datatypes and toward best practices.

<img width="957" alt="Screenshot 2023-08-15 at 11 41 46 AM" src="https://github.com/galaxyproject/galaxy/assets/216771/ecb4c592-8085-4077-aa98-083844bbb629">

I think this supports best practices and helps new users who might encounter problems without eliminating valid use cases and maintaining reproducibility. An alternative approach of just dropping the types from upload has been recommend by @martenson though and I think it is a proposal worth considering - I am just not there.

## How to test the changes?
(Select all options that apply)
  1. Try uploading a fastq explicitly and see message in screenshot.

## License
